packages/cli/index.js

```diff
@@ -1,18 +1,17 @@
 #!/usr/bin/env node
 'use strict';
 
-const { promisify } = require('util');
 const { dirname } = require('path');
 
-const findUp = require('find-up');
-const resolve = promisify(require('enhanced-resolve'));
+const { sync: findUp } = require('find-up');
+const { sync: resolve } = require('enhanced-resolve');
 
-findUp('package.json')
-  .then(pkgFile =>
-    resolve(dirname(pkgFile), '@untool/yargs').then(path => require(path).run())
-  )
-  .catch(error => {
-    // eslint-disable-next-line no-console
-    console.error(error.stack ? error.stack.toString() : error.toString());
-    process.exit(1);
-  });
+try {
```

Comment:
That is the reason why I added a `try/catch` block inside `run()`: I want `run()` to handle error logging consistently because I consider the `cli` package entirely optional (possibly bordering obsolete).